Backend/app/rider-platform/rider-app/Main/src/SharedLogic/CallBPPInternal.hs

```diff
@@ -108,7 +108,8 @@ disputeCancellationDues apiKey internalUrl merchantId phoneNumber countryCode me
 
 data CancellationDuesDetailsRes = CancellationDuesDetailsRes
   { customerCancellationDues :: HighPrecMoney,
-    disputeChancesUsed :: Int
+    disputeChancesUsed :: Int,
+    canBlockCustomer :: Bool
```

Comment:
Not a backward compatible change.

Reply:
why? It is not being used by UI as of now and an extra field is added in the response